pkg/options/types.go

```diff
@@ -106,56 +105,21 @@ func (r *ResourceSet) Type() string {
 }
 
 // NodeType represents a nodeName to query from.
-type NodeType map[string]struct{}
+type NodeType string
```

Comment:
Because this is no longer a complex type should we just remove this type alias so we don't have to cast in other places?

Reply:
This seems necessary since there seem to be methods defined on it below.